packages/cli/src/api/fixtures/collect-typescript-jsx/jsx-syntax.jsx

```diff
@@ -5,34 +5,38 @@ const jsx = <div>Hello!</div>
 @Decorator()
 export class TestDecorator {
   @Decorator()
-  prop;
+  prop
 
   @Decorator()
-  method() {};
+  method() {}
 }
 
+export
+@Decorator()
+class TestDecoratorAfterExport {}
+
 class A {
   // classProperties
-  b = 1;
+  b = 1
 
   // classPrivateProperties
-  #b = 1;
+  #b = 1
 }
 
 // dynamicImport
-import('./guy').then(a)
+import("./guy").then(a)
```

Comment:
Prettier change the formatting on save / before commit actions even if this files added as ignored 😬